src/utils/vacation-days-utils.tsx

```diff
@@ -1,38 +1,45 @@
 import { Grid, Typography } from "@mui/material";
 import { theme } from "../theme";
-import type { Person } from "../generated/client";
 import strings from "../localization/strings";
 import type { User } from "src/generated/homeLambdasClient";
 
 /**
  * Display persons vacation days
  *
- * @param Person timebank person
+ * @param User KeyCloak user
  */
-export const renderVacationDaysTextForCard = (person: Person) => {
-  const spentVacationsColor =
-    person && person.spentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
+export const renderVacationDaysTextForCard = (user: User) => {
+  // const spentVacationsColor =
+  //   user && user.spentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
+  //
+  // const unspentVacationsColor =
+  //   user && user.unspentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
 
-  const unspentVacationsColor =
-    person && person.unspentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
+  //FIXME: Deal with the spent and unspent vacations
+  const spentVacationsColor = theme.palette.error.main;
+  const unspentVacationsColor = theme.palette.error.main;
 
-  if (person) {
+  if (user) {
     return (
       <Grid>
         <Grid container spacing={2} alignItems="center">
           <Grid item xs={6}>
             {strings.vacationsCard.spentVacations}
           </Grid>
           <Grid item xs={6}>
-            <Typography color={spentVacationsColor}>{person.spentVacations}</Typography>
+            <Typography color={spentVacationsColor}>
+              {/*{user.spentVacations}*/}
```

Comment:
Is this commented out because user no longer contains spentVacations? If so we should create a task to add spent vacations to the user- lets discuss this is in the tasking session later today.